content/en/blog/2023/announcing-notation-rc4.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Notation v1.0.0-RC.4 is available!
+author:  "Notation Release Team"
+date: 2023-04-21
+draft: false
+---
+
+The Notation maintainers are pleased to announce the releases of Notation CLI v1.0.0-rc.4, notation-go v1.0.0-rc.4, and notation-core-go v1.0.0-rc.3. This blog walks you through the major updates of this release.
+
+## What's new
+
+This release adds the following significant enhancements:
+
+- Support validating certificate revocation with [Online Certificate Status Protocol](https://datatracker.ietf.org/doc/html/rfc6960) (OCSP)
+- Introduce switch `NOTATION_EXPERIMENTAL=1` to enable experimental features
+- Introduce new CLI command `notation policy` to simplify trust policy configuration
+- Support OCI distribution referrers API
+- Introduce signing, listing and verification with [OCI image layout](https://github.com/opencontainers/image-spec/blob/v1.0/image-layout.md) as experimental feature
+
+### Support validating Certificate revocation with OCSP
+
+OCSP stands for Online Certificate Status Protocol. It is a protocol used to check the revocation status of digital certificates. When a certificate is revoked, it means that it is no longer valid and should not be trusted. To enforce validating certificate revocation, the verification level of trust policy configuration should be set to `strict` as following. Once this is done, the CLI command `notation verify` can be used to validate the signatures including checking certificate revocation status with OCSP. The verification process will fail if any certificate in the certificate chain has been revoked.
+
+```json
+"signatureVerification": {
+    "level" : "strict" 
+}
+```
+
+### Introduce switch `NOTATION_EXPERIMENTAL=1` to enable experimental features
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. You can now enable experimental features in Notation CLI by setting the `NOTATION_EXPERIMENTAL` environment variable to `1`. Here's an example of how to set the environment variable in Linux or macOS:
+
+```shell
+export NOTATION_EXPERIMENTAL=1
+```
+
+And here's an example of how to set the environment variable in Windows PowerShell:
+
+```powershell
+$env:NOTATION_EXPERIMENTAL=1
+```
+
+Once you've set the environment variable, you can use Notation with experimental features enabled. Here is the list of experimental features that Notation introduced:
+
+- Store signatures using artifact manifest (Require Registry support)
```

Comment:
I thought had this already without the experimental feature. Did we redo the existing "artifact manifest" feature inside the experimental feature?

Reply:
Yes, This one is not controlled by experimental environment variable in previous release, but it is in RC-4

Reply:
In that case it becomes somewhat of a ux change. We should let the users know that it is a change from Rc-3 behaviour.

Reply:
Called it out in the section of "what's new"